packages/mongodb-verified-answers/src/commands/import.ts

```diff
@@ -0,0 +1,72 @@
+import { CommandModule } from "yargs";
+import { promises as fs } from "fs";
+import { MongoClient } from "mongodb";
+import {
+  assertEnvVars,
+  CORE_ENV_VARS,
+  makeOpenAiEmbedder,
+  OpenAIClient,
+  AzureKeyCredential,
+} from "mongodb-rag-core";
+import { parseVerifiedAnswerYaml } from "../parseVerifiedAnswersYaml";
+import { importVerifiedAnswers } from "../importVerifiedAnswers";
+
+import "dotenv/config";
+
+const commandModule: CommandModule<
+  Record<string, unknown>,
+  ImportCommandArgs
+> = {
+  command: "pages <path>",
```

Comment:
```suggestion
  command: "import <path>",
```

Looks like this may have been left over from ingest?